parameters.d/osg34-stashcache.yaml

```diff
@@ -0,0 +1,47 @@
+###################
+# OSG 3.4 StashCache tests
+# File format documention:
+# https://github.com/opensciencegrid/vm-test-runs#running-osg-test-in-vm-universe
+###################
+
+platforms:
+  - centos_7_x86_64
+  - sl_7_x86_64
+
+sources:
+  ###################
+  # Format:
+  # [<Github account>:<osg-test branch>;] <OSG ver>; <REPO 1, REPO 2...REPO N> [> <Update OSG ver>/<Update REPO 1, REPO 2...REPO N>]
+  # Example:
+  # Run osg-test with packages from 3.2-release and 3.2-testing that are then upgraded to 3.3-testing and 3-3-upcoming-testing:
+  # 3.2; osg, osg-testing > 3.3/osg-testing, osg-upcoming-testing
+  ###################
+  # - opensciencegrid:master; 3.4; osg
```

Comment:
Why exclude the release repo?

Reply:
Not sure the version of SC in release works with the tests, but let me give it a shot...

Reply:
Looks like it works so I added it.